functions/lib/lib.js

```diff
@@ -0,0 +1,79 @@
+'use strict'
+
```

Comment:
can we call this file something more meaningful like `azure-storage-helper.js`?

in `index.js` above it is not clear what the intention is from requiring `lib/lib`

Reply:
Seems fair enough.  👍